regression/cbmc-library/syscall-01/main.c

```diff
@@ -0,0 +1,13 @@
+#include <sys/syscall.h>
+
+#include <assert.h>
+#include <errno.h>
+#include <unistd.h>
+
+int main()
+{
+  long int rc;
```

Comment:
should this also use the __APPLE__ env to change the type as in the definition ?